.github/workflows/go.yml

```diff
@@ -14,15 +14,30 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
 
-    - name: Set up Go
-      uses: actions/setup-go@v3
-      with:
-        go-version: 1.19
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
 
-    - name: Build
-      run: go build -v cmd/gitlab-review-bot/main.go
+      - name: Build
+        run: go build -v cmd/gitlab-review-bot/main.go
 
-    - name: Test
-      run: go test -v ./...
+      - name: Test
+        run: go test -v ./...
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
As per my replies in task I created `DOCKER_HUB_USERNAME`, not `secrets.DOCKERHUB_USERNAME`.

Same for token.

Reply:
Ok